File: php/examples/post_auth_forgot_password.php
<?php
require_once 'base_request.php';

// Example of POST /auth/forgot-password

$debug = true;
$email = $_ENV['EMAIL'];
// $subdomain = $_ENV['SUBDOMAIN'];    // optional
$website_id = $_ENV['WEBSITE_ID'];  // optional

$request_data = array(
    'email_address' => $email,
    'website_id' => $website_id
);

$request_params = '';

$json_body = json_encode($request_data);

send_request('POST', '/auth/forgot-password', $request_params, $json_body);

File: csharp/examples/PostAuthForgotPassword.cs
using System;

namespace Examples.examples
{
    // Example of POST /auth/forgot-password
    public class PostAuthForgotPassword : BaseRequest
	{
        private string _email;
        private string _subdomain;  // optional
        private string _website_id; // optional

        public PostAuthForgotPassword()
		{
            _email = Environment.GetEnvironmentVariable("EMAIL");
            _subdomain = Environment.GetEnvironmentVariable("SUBDOMAIN");
            _website_id = Environment.GetEnvironmentVariable("WEBSITE_ID");

            Console.WriteLine($"PostAuthForgotPassword: _email={_email} _subdomain={_subdomain} _website_id={_website_id}");
        }

        public override void SendRequest()
        {
            var body = Newtonsoft.Json.JsonConvert.SerializeObject(new
            {
                email_address = _email,
                // subdomain: _subdomain,
                website_id = _website_id
            });

            PostRequest("/auth/forgot-password", body);
        }
    }
}